provider/aws.go

```diff
@@ -380,13 +404,56 @@ func (p *AWSProvider) submitChanges(changes []*route53.Change, zones map[string]
 					},
 				}
 
+				successfulChanges := 0
+
 				if _, err := p.client.ChangeResourceRecordSets(params); err != nil {
-					log.Errorf("Failure in zone %s [Id: %s]", aws.StringValue(zones[z].Name), z)
+					log.Errorf("Failure in zone %s [Id: %s] when submitting change batch", aws.StringValue(zones[z].Name), z)
 					log.Error(err) //TODO(ideahitme): consider changing the interface in cases when this error might be a concern for other components
-					failedUpdate = true
+
+					if len(b) > 1 {
+						log.Error("Trying to submit changes one-by-one instead")
```

Comment:
looks more like a debug log to me, or delete it

Reply:
Hm are you sure about this? If there is a transaction failing, in my opinion this should output more verbose logs than debug because it needs a manual intervention to fix it.